tools/DevCheck.ps1

```diff
@@ -39,26 +54,41 @@
 #>
 
 Param(
-    [Switch]$CheckAll=$false,
+    [String]$CertPassword=$null,
```

Comment:
Does the `CertPassword` get used by the caller? If not, can it be a random string and then the codebase wouldn't have even a test password stored. Internally, we use a pattern like this for generating certificates to sign some test. I can share that with you over Teams/email.

Reply:
Yes. We need the password saved to disk for later reuse.

*.pfx are password protected. Using them (.e.g to make a signed .msix) requires the password. If by 'a random string' you mean 'a defined constant' that wouldn't be any better than no password. Also, some of this work requires admin privilege -- but normal development doesn't. So we do the work here to create the cert (.cer, .pfx) and save the password for later use.

The build pipeline specifies a value because it has to (after all, there's no human to do 1-time setup). See `build\AzurePipelinesTemplates\WindowsAppSDK-BuildProject-Steps.yml`. But that's done on the build machine during a build, gets stashed in the build environment, used during the build and then torched when the build's over, never copied off that machine.

And that's a constant as an interim thing (crawl, walk, run). I'd like to change it to a random number (for instance, `$CertPassword = New-Guid`) but I have questions about the build pipelines (e.g. build machine reuse) so best handled as a later change. @EricJohnson327 this also poses questions with the build work you're doing. I was planning to bring it up after we cleared some work from our plates.